tools/test-backend

```diff
@@ -65,6 +65,27 @@ if __name__ == "__main__":
                       default=False,
                       help="Show which tests are slowest.")
 
+    from version import PROVISION_VERSION
+    version_file = 'var/provision_version'
+    try:
+        version = open(version_file).read().strip()
+    except:
+        print('''
+            You must populate %s by running
+            provision.py. Or, if you know you have provisioned
+            recently, run this command:
+
+            echo %s > %s''' % (version_file, PROVISION_VERSION, version_file))
+        sys.exit(1)
+
+    if version != PROVISION_VERSION:
+        print('''
+            The value in %s (%s) suggests you
+            are out of sync in provisioning (see PROVISION_VERSION
+            in version.py).  Please provision before running
```

Comment:
We should mention (0) this is usually caused by rebasing on top of upstream/master to pull in other developer's changes that added new dependencies or whatnot, (1) if you're behind you may want to rebase and (2) a `--force` flag to ignore and just run it.